configs/STELLARF4V2/config.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU STM32F405
+#define BOARD_NAME STELLARF4
+#define MANUFACTURER_ID STBE
+
+#define TARGET_BOARD_IDENTIFIER "STF4"
+#define USBD_PRODUCT_STRING     "StingBee Stellar F4 V2"
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42605
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42605
+#define USE_GYRO_SPI_ICM42688P
+#define USE_GYRO_CLKIN
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_SDCARD
```

Comment:
Not used
```suggestion
```

Reply:
I was thinking about the future, you can use SD NAND in an LGA8 package

https://www.lcsc.com/datasheet/lcsc_datasheet_2304140030_XTX-XTSD04GLGEAG_C558839.pdf

I'd better check it in the prototype fits.